composer.json

```diff
@@ -21,12 +21,12 @@
         "doctrine/common": "~2.0"
     },
     "require-dev": {
-        "symfony/symfony": "^2.7.1|^3.3|^4.0",
-        "symfony/phpunit-bridge": "^2.7|^3.0|^4.0",
+        "symfony/symfony": "^2.7.1|^3.3",
```

Comment:
This change is good, but unfortunately we have previous ones that will have those wrong constraints... Can we do nothing about it?

Reply:
You're right. Unfortunately, I don't have any solution for previous versions. We can't delete them since it may break existing tests, CI, etc. It would be better to mark previous releases as deprecated but I don't think that we can do that with Composer.

Reply:
It seems not: https://github.com/composer/composer/issues/7194